.vsts-dotnet.yml

```
@@ -8,176 +8,189 @@ trigger:
 # variables:
 #   SignType: real
 #   SkipApplyOptimizationData: false
+stages:
+- stage: build
+  displayName: Build and Test
```

Nit:
```suggestion
  displayName: Build
```

we don't test in this build. We can think about testing in another stage, now that it won't block publish.

---

Copy paste from Roslyn. Let me fix this real quick.